lib/src/corsheaders.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io' show HttpHeaders;
+
+extension CORSHeadersExt on HttpHeaders {
+  static const corsAllowedOriginsHeader = CORSHeaders;
```

Comment:
this need to be instanciate with `()`

Reply:
if the idea is to have the same thing as `generalHeaders` in the sdk.
we must do the list ourself

Reply:
It doesn't have to be Instantiated because all of the strings are `static const`, if I were to instantiate them there, then I'd have to change every field in `CORSHeaders` to be `final`